index.html

```diff
@@ -0,0 +1,346 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="IE=edge"
+    />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Project1</title>
+  </head>
+  <body class="bg-secondary">
+    <div class="container-fluid">
+      <!-- logo & hamburger menu -->
+      <header class="logo-menubar d-flex1">
+        <img src="./images/logo.svg" alt="logo" />
+        <img
+          class="menu-icon"
+          src="./images/menu.svg"
+          alt=""
+        />
+      </header>
+      <!-- end of logo & hamburger menu -->
+      <!-- main body start -->
+      <main class="main display-flex2">
+        <div class="intro">
+          <h1>
+            I’m Balqees Ibrahim Glad to see you!
+          </h1>
+          <p>
+            I’m a software developer! I can help
+            you build a product feature or
+            website. Look through some of my work
+            and experience! If you like what you
+            see and have a project you need coded,
+            don’t hesitate to contact me.
+          </p>
+        </div>
+        <a class="connect" href="#"
+          >let's connect</a
```

Comment:
- [ ] Awesome having links to connect. `a` tag are for only links. Here are links as well headings. So let's wrap them inside another heading tag